src/messaging/ExchangeMgr.cpp

```diff
@@ -352,7 +352,9 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
             ChipLogError(ExchangeManager, "OnMessageReceived failed, err = %" CHIP_ERROR_FORMAT,
                          CHIP_ERROR_INVALID_MESSAGE_TYPE.Format());
             ec->Close();
+
```

Comment:
Why the random whitespace changes?

Reply:
Will remove. I may have added and deleted something earlier and these lingered.